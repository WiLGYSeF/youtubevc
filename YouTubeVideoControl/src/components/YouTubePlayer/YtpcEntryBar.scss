.entry-bar {
  align-items: center;
  background-color: #f00;
  display: flex;
  height: 2px;
  width: 100%;

  .arrow {
    border: solid 6px;
    height: 0;
    width: 0;
    border-color: transparent transparent transparent #f00;
    z-index: 10;
  }
}
