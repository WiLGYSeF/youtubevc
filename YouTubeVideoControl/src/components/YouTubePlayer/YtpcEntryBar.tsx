import React from 'react';

import './YtpcEntryBar.scss';

function YtpcEntryBar() {
  return (
    <div className="entry-bar">
      <div className="arrow" />
    </div>
  );
}

export default YtpcEntryBar;
