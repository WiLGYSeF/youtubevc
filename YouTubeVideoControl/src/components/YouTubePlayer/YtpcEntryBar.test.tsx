import React from 'react';
import renderer from 'react-test-renderer';

import YtpcEntryBar from './YtpcEntryBar';

describe('YtpcEntryBar', () => {
  it('renders', () => {
    const component = renderer.create(<YtpcEntryBar />);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
