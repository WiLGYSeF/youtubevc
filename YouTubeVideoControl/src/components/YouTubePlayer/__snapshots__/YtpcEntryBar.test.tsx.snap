// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`YtpcEntryBar renders 1`] = `
<div
  className="entry-bar"
>
  <div
    className="arrow"
  />
</div>
`;
